### PR TITLE
Added display_name attribute to all files/folders in endpoint

### PR DESCRIPTION
**Pull request by Majkluss**

### CODE REVIEW

File: prusa/link/web/files.py [hunk 1]
```diff
@@ -84,34 +85,41 @@ def api_file_info(req, storage, path=None):
...
         result.update(fill_printfile_data(path, os_path, storage))
 
     # Fill specific data for firmware file
-    # elif file_type is FileType.FIRMWARE.value:
+    elif file_type is FileType.FIRMWARE.value:
+        result.update(fill_file_data(path, storage))
 
     # Fill specific data for other file
-    # elif file_type is FileType.FILE.value:
+    elif file_type is FileType.FILE.value:
```
Comment (ondratu) [inline prusa/link/web/files.py:121]: What happen if is there another `file_type`? The better solution is expect that is `FileType.File` so use else here.

File: prusa/link/web/lib/files.py [hunk 2]
```diff
@@ -89,10 +115,15 @@ def fill_printfile_data(path: str, os_path: str, storage: str):
         meta.load_from_path(path)
         result['refs'] = sdcard_refs()
 
-    result['meta'] = meta.data
-    result['meta']['estimated_print_time'] = estimated_to_seconds(
+    estimated = estimated_to_seconds(
         meta.data.get('estimated printing time (normal mode)', ''))
-    result['display_name'] = basename(path)
+
```
Comment (ondratu) [inline prusa/link/web/lib/files.py:120]: i have to discuss it with Michal and Ilja first

**Review by ondratu — CHANGES_REQUESTED** [review]

File: prusa/link/web/lib/files.py (hunk 2 above)
Comment (Majkluss) [reply prusa/link/web/lib/files.py:120]: Yes, this is just a preparation, I can change it

File: prusa/link/web/files.py (hunk 1 above)
Comment (Majkluss) [reply prusa/link/web/files.py:121]: Makes sense, I'll fix it.

File: prusa/link/web/lib/files.py [hunk 3]
```diff
@@ -89,10 +115,15 @@ def fill_printfile_data(path: str, os_path: str, storage: str):
...
+    if simple:
+        result['filament_type'] = meta.data.get('filament_type')
```
Comment (ondratu) [inline prusa/link/web/lib/files.py:122]: please no any other data

**Review by ondratu — CHANGES_REQUESTED** [review]

**Review by ondratu — APPROVED** [review]

**Pull request merged**